lib/generators/open_stax/rescue_from/install/install_generator.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails/generators'
+
+module OpenStax
+  module RescueFrom
+    class InstallGenerator < Rails::Generators::Base
+      source_root File.expand_path("../templates", __FILE__)
+
+      def copy_templates
+        copy_file "any.html.erb", "app/views/errors/any.html.erb"
+      end
+
+      def copy_initializer
+        copy_file "rescue_from.rb", "config/initializers/rescue_from.rb"
+      end
+
+      def configure_environments
+        test_file = "test.rb"
+        test_middleware = <<-TEST
+  config.middleware.use ExceptionNotification::Rack, email: {
```

Comment:
It would be **really** great to be able to hide the ExceptionNotification configuration totally within the gem if possible.  Seems non-ideal to install a gem that puts some other gem's settings into the app's config files.

Do these links give us a way to have the configuration for ExceptionNotification to be hidden?
* http://edgeapi.rubyonrails.org/classes/Rails/Engine.html#class-Rails::Engine-label-Middleware+stack  ?
* http://stackoverflow.com/a/6901382